app/assets/v2/js/shared.js

```diff
@@ -656,21 +656,21 @@ const showChoices = (choice_id, selector_id, choices) => {
   });
 };
 
-var retrieveIssueDetails = function() {
-  var ele = $('input[name=issueURL]');
-  var target_eles = {
+const retrieveIssueDetails = function() {
+  let ele = $('input[name=issueURL]');
+  let target_eles = {
```

Comment:
Please try not to make changes to the codebase with automated tools that fix style, etc. - it makes it more difficult to review the code. These changes aren't too bad but as a general rule, minimize your changes to the functionality described in the PR/issue.